src/main/resources/test_files/runtime/FunctionBinaries/FunctionBase64BinaryError2.jq

```diff
@@ -1,4 +1,4 @@
-(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:2:COLUMN:13:" :)
+(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:2:COLUMN:0:" :)
```

Comment:
The fix of swapping the two iterators to disentangle them may possibly help get the right location hint back, because then each iterator works standalone again.